editor/src/components/navigator/navigator-item/navigator-item.tsx

```diff
@@ -272,6 +275,69 @@ function useIsProbablyScene(path: ElementPath): boolean {
   )
 }
 
+const isHiddenConditionalBranchSelector = createSelector(
+  (store: MetadataSubstate, _elementPath: ElementPath, parentPath: ElementPath) =>
+    MetadataUtils.findElementByElementPath(store.editor.jsxMetadata, parentPath),
+  (_store: MetadataSubstate, elementPath: ElementPath, _parentPath: ElementPath) => elementPath,
+  (_store: MetadataSubstate, _elementPath: ElementPath, parentPath: ElementPath) => parentPath,
+  (
+    parent: ElementInstanceMetadata | null,
+    elementPath: ElementPath,
+    parentPath: ElementPath,
+  ): boolean => {
+    const conditional = asConditional(parent)
+    if (conditional == null) {
+      return false
+    }
+    const flag = getConditionalFlag(conditional)
+    return flag == false
```

Comment:
```suggestion
    return getConditionalFlag(conditional)
```

Reply:
🤦 🤯 good spot!
doing a full commit instead of the suggestion to simplify the related code

Reply:
actually no, this won't work: the existing code is correct because you need to flip the check for the _other_ branch that is not active, while checking the current one.